deploy/olm-catalog/kogito-operator/0.13.0/kogito-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -101,9 +101,9 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "false"
-    containerImage: quay.io/kiegroup/kogito-cloud-operator:0.13.0
+    containerImage: quay.io/kiegroup/kogito-cloud-operator:0.10.0
```

Comment:
Is this intentional change?

Reply:
I don't think so!

Reply:
https://github.com/kiegroup/kogito-cloud-operator/blob/f4c3e4f1b7c58df829b2ebee9b99aba911bfa02e/deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml#L104

Reply:
updating the bump-version.sh script to update that one